dev/how-to/maintain/identify-slow-queries.md

```diff
@@ -2,17 +2,12 @@
 title: Identify Slow Queries
 summary: Use the slow query log to identify problematic SQL statements.
 category: how-to
+aliases: ['/docs/sql/slow-query/']
 ---
 
 # Identify Slow Queries
 
-The slow query log is a record of SQL statements that took a long time to perform.
-
-A problematic SQL statement can increase the pressure on the entire cluster, resulting in a longer response time. To solve this problem, you can use the slow query log to identify the problematic statements and thus improve the performance.
-
-## Obtain the log
-
-In TiDB, the statements whose execution time exceeds [slow-threshold](/reference/configuration/tidb-server/configuration-file.md#slow-threshold) are individually output to [slow-query-file](/reference/configuration/tidb-server/configuration-file.md#slow-query-file) by default, the format of the slow log is compatible, and the slow log file can be directly analyzed with `pt-query-digest`. `slow-threshold` can be modified by the configuration file, which is set to 300ms by default. `slow-query-file` is set to `tidb-slow.log` by default.
+To help users identify slow queries, analyze and improve performance of SQL execution, TiDB outputs statements whose execution time exceeds [slow-threshold](/reference/configuration/tidb-server/configuration-file.md#slow-threshold) (The default value is 300 milliseconds) to [slow-query-file](/reference/configuration/tidb-server/configuration-file.md#slow-query-file) (The default value is "tidb-slow.log").
 
 ## Usage example
```

Comment:
This part is not aligned with https://github.com/pingcap/docs-cn/pull/1752. Please make corresponding changes in the code block.